Resources/Private/Templates/Core11/DataSheetImport/Upload.html

```diff
@@ -0,0 +1,99 @@
+<html
+    xmlns:f="http://typo3.org/ns/TYPO3/Fluid/ViewHelpers"
+    xmlns:be="http://typo3.org/ns/TYPO3/CMS/Backend/ViewHelpers"
+    xmlns:core="http://typo3.org/ns/TYPO3/CMS/Core/ViewHelpers"
+    xmlns="http://www.w3.org/1999/xhtml"
+    lang="en"
+    f:schemaLocation="https://fluidtypo3.org/schemas/fluid-master.xsd"
+    data-namespace-typo3-fluid="true"
+>
+<f:layout name="Default" />
+
+<f:section name="Content">
+    <h1><f:translate key="module_name" extensionName="xlsimport" /> - <f:translate key="prepare" extensionName="xlsimport" /></h1>
+    <f:be.infobox title="{f:translate(key:'prepareinfo', extensionName: 'xlsimport')}" state="-1">
```

Comment:
inline notation here is okay as used for a tag attribute / other viewhelper attribute